docs/using-manila-csi-plugin.md

```diff
@@ -61,35 +61,46 @@ Optionally, a custom certificate may be sourced via `os-certAuthorityPath` (path
 
 ## Deployment
 
-The Manila CSI driver deals with the Manila service only. All node-related operations (attachments, mounts) are performed by a dedicated CSI Node Plugin, to which all Node Service RPCs are forwarded. This means that the operator is expected to already have a working deployment of that dedicated CSI Node Plugin.
+The CSI Manila driver deals with the Manila service only. All node-related operations (attachments, mounts) are performed by a dedicated CSI Node Plugin, to which all Node Service RPCs are forwarded. This means that the operator is expected to already have a working deployment of that dedicated CSI Node Plugin.
 
 A single instance of the driver may serve only a single Manila share protocol. To serve multiple share protocols, multiple deployments of the driver need to be made. In order to avoid deployment collisions, each instance of the driver should be named differently, e.g. `csi-manila-cephfs`, `csi-manila-nfs`.
 
-### Kubernetes 1.13+
+### Kubernetes 1.15+
 
-Required feature gates: `CSIDriverRegistry=true`, `CSINodeInfo=true`. Snapshots require `VolumeSnapshotDataSource=true` feature gate.
+Snapshots require `VolumeSnapshotDataSource=true` feature gate.
+
+The deployment consists of two main components: Controller and Node plugins along with their respective RBACs. Controller plugin is deployed as a StatefulSet and runs CSI Manila, [external-provisioner](https://github.com/kubernetes-csi/external-provisioner) and [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter). Node plugin is deployed as a DaemonSet and runs CSI Manila and [csi-node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar).
+
+**Deploying with Helm**
+
+This is the preferred way of deployment because it greatly simplifies the difficulties with managing multiple share protocols.
+
+CSI Manila Helm chart is located in `examples/manila-csi-plugin/helm-deployment`.
+
+First, modify `values.yaml` to suite your environment, and then simply install the Helm chart with `$ helm install helm-deployment`.
```

Comment:
We could add a note here about naming, due to the way the resources in our manifests are named, the helm release name should be 12 characters or less? Alternatively, we could find a way to shorten the resource names by dropping the "plugin" in the names? just a thought..